docs/example.ipynb

```diff
@@ -105,15 +105,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**one or two sentences describing what this function does in the context of usage and functionality**"
+    "The frequency_vectorizer calculates the frequency of each word in a list of text documents. This function is useful for transforming text data into a feature matrix, and word frequency matrix.\n",
```

Comment:
Small modification.
```suggestion
    "The `frequency_vectorizer` calculates the frequency of each word in a list of text documents to capture the significance of each word in each document. This function is useful for transforming text data into a feature matrix (word frequency matrix) that is to be used for machine learning.\n",
```

Reply:
Thanks, looks good!

Reply:
I meant to add your suggestions actually.